.github/workflows/build.yml

```diff
@@ -91,6 +99,29 @@ jobs:
         docker push antrea/antrea-ubi:latest
         docker push antrea/antrea-agent-ubi:latest
         docker push antrea/antrea-controller-ubi:latest
+    - name: Install Kind
```

Comment:
It's a bit strange to test the image _after_ pushing it to the registry, but as long as we address #6031 promptly that works for me

Reply:
Thanks. I can follow up with https://github.com/antrea-io/antrea/issues/6031 after this PR.